spread.yaml

```diff
@@ -57,6 +57,24 @@ backends:
                 username: ubuntu
                 password: ubuntu
 
+    external-ubuntu-core:
+        type: adhoc
+        environment:
+            SPREAD_ADHOC_UC_ADDRESS: "$(HOST: echo ${SPREAD_ADHOC_UC_ADDRESS:-localhost:8022})"
+        allocate: |
+            ADDRESS $SPREAD_ADHOC_UC_ADDRESS
+        systems:
+            # FIXME: remove the ubuntu-core-16.04-64 dummy system once the runner handles tasks without systems assigned
+            - ubuntu-core-16.04-64:
+                username: test
+                password: ubuntu
+            - ubuntu-core-16-device-amd64:
```

Comment:
Ok, Samuele is the right person to explain this, for the record this is the error I was getting when naming the system `ubuntu-core-16-64` and executing against a running amd64 instance:
```
2016/09/09 16:46:16 Error executing adhoc-ubuntu-core:ubuntu-core-16-64:tests/main/ack : 
-----
mesg: ttyname failed: Inappropriate ioctl for device
+ echo 'Ack when missing matching key fails'
Ack when missing matching key fails
+ snap ack developer1.account
error: cannot assert: assert failed: no matching public key "XCIC_Wvj9_hiAt0b10sDon74oGr3a6xGODkMZqrj63ZzNYUD5N87-ojjPoeN7f1Y" for signature by "testrootorg"
+ echo 'Ack the test store key'
Ack the test store key
+ snap ack testrootorg-store.account-key
error: cannot assert: assert failed: no matching public key "hIedp1AvrWlcDI4uS_qjoFLzjKl5enu4G2FYJpgB3Pj-tUzGlTQBxMBsBmi-tnJR" for signature by "testrootorg"
```